test/e2e/l7networkpolicy_test.go

```diff
@@ -290,40 +295,70 @@ func testL7NetworkPolicyTLS(t *testing.T, data *TestData) {
 	_, err := data.podWaitForIPs(defaultTimeout, clientPodName, data.testNamespace)
 	require.NoError(t, err, "Expected IP for Pod '%s'", clientPodName)
 
-	l7ProtocolAllowsGoogle := []crdv1beta1.L7Protocol{
+	serverPodName := "test-l7-tls-server"
+	serverPodLabels := map[string]string{"test-l7-tls-e2e": "server"}
+	// Start an HTTPS server with the agnhost image build-in certificate.
+	cmd := []string{"/agnhost", "netexec", "--http-port=443", "--tls-cert-file=/localhost.crt", "--tls-private-key-file=/localhost.key"}
+	require.NoError(t, NewPodBuilder(serverPodName, data.testNamespace, agnhostImage).OnNode(nodeName(0)).WithCommand(cmd).WithLabels(serverPodLabels).Create(data))
+	podIPs, err := data.podWaitForIPs(defaultTimeout, serverPodName, data.testNamespace)
+	require.NoError(t, err, "Expected IP for Pod '%s'", serverPodName)
+	serverIPs := podIPs.AsSlice()
+
+	testSNISuffixAlfa := "alfa.test.l7.tls"
+	testSNISuffixBravo := "bravo.test.l7.tls"
+	allTestSNIs := make(map[string][]string)
```

Comment:
This is not needed if you just pass `serverIPs` and `serverName` to `probeL7NetworkPolicyTLS`.

Reply:
Removed